web/write.php

```diff
@@ -279,20 +279,32 @@ function buildWriteForm($options) {
 
     $form->addElement('html', '</div>'); # close div.row
 
-    $form->addElement("html", '<fieldset class="last"><legend>Ready?</legend>');
+    $form->addElement("html", '<fieldset class="last">');
 
         add_all_variables_hidden($form, $fyr_values, $options);
 
-        $preview_text = cobrand_preview_text($cobrand);
-        if (!$preview_text) {
-            $preview_text = "When you’re ready to send your message click the button below. You’ll get a chance to read through it again to check for problems before we send it.";
```

Comment:
You don't think it's worth keeping this message or something like it after this new text? To some people it might not be clear what to do...

Reply:
As soon as we get our first user support message from a person not knowing what to do after they’ve filled in their details, I’ll add that sentence back in 😉